File: targets/latest-items-widget/DirectusWidgetConstants.swift
import Foundation

/// Shared Directus flow / asset constants for widget extensions (mirrors Android `DirectusWidgetConstants`).
enum DirectusWidgetConstants {
  static let flowBlockTypeLatestItems = "latest-items"

  /// Directus asset transform: raster output so `UIImage` can decode (SVG often ignores params).
  static let directusAssetRasterQuery = "width=64&height=64&fit=cover&format=png"
}

File: android/app/src/main/java/com/martijnmichel/directusexpo/widget/directus/DirectusWidgetConstants.kt
package com.martijnmichel.directusexpo.widget.directus

object DirectusWidgetConstants {
  /** Flow `data[]` block type for latest-items schema. */
  const val FLOW_BLOCK_TYPE_LATEST_ITEMS = "latest-items"

  /** Directus asset transform: prefer raster output for BitmapFactory decode. */
  const val ASSET_RASTER_QUERY = "width=64&height=64&fit=cover&format=png"
}
